### ISSUE TITLE
Build issue on astrolink4 with CircleCI

### ISSUE THREAD

**Issue by thx8411** [indilib/indi-3rdparty#196]
Hi,

There is a build error on astrolink4 with CircleCI :

It's actually a warning,  but "_cc1plus: all warnings being treated as errors_" kills the job.

<<
[ 12%] Building CXX object indi-astrolink4/CMakeFiles/indi_astrolink4.dir/indi_astrolink4.cpp.o
/root/project/indi-astrolink4/indi_astrolink4.cpp: In member function 'virtual IPState IndiAstrolink4::MoveAbsFocuser(uint32_t)':
/root/project/indi-astrolink4/indi_astrolink4.cpp:694:41: error: comparison of unsigned expression < 0 is always false [-Werror=type-limits]
             if((targetTicks + backlash) < 0 || (targetTicks + backlash) > FocusMaxPosN[0].value)
                ~~~~~~~~~~~~~~~~~~~~~~~~~^~~
cc1plus: all warnings being treated as errors
indi-astrolink4/CMakeFiles/indi_astrolink4.dir/build.make:62: recipe for target 'indi-astrolink4/CMakeFiles/indi_astrolink4.dir/indi_astrolink4.cpp.o' failed
make[2]: *** [indi-astrolink4/CMakeFiles/indi_astrolink4.dir/indi_astrolink4.cpp.o] Error 1
CMakeFiles/Makefile2:236: recipe for target 'indi-astrolink4/CMakeFiles/indi_astrolink4.dir/all' failed
make[1]: *** [indi-astrolink4/CMakeFiles/indi_astrolink4.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2
>>

Best regards,

Blaise

**Comment by pawel-soja** [comment]
Fixed in #299

**Comment by ggtux** [comment]
unsubscribe

Von meinem iPhone gesendet
